src/VisualStudio/CSharp/Impl/Options/Formatting/CodeStylePage.cs

```diff
@@ -11,7 +19,502 @@ internal class CodeStylePage : AbstractOptionPage
     {
         protected override AbstractOptionPageControl CreateOptionPage(IServiceProvider serviceProvider)
         {
-            return new GridOptionPreviewControl(serviceProvider, (o, s) => new StyleViewModel(o, s));
+            return new GridOptionPreviewControl(serviceProvider, (o, s) => new StyleViewModel(o, s), GetCurrentEditorConfigOptionsString);
+        }
+
+        internal static string GetCurrentEditorConfigOptionsString(OptionSet optionSet)
+        {
+            var editorconfig = new StringBuilder();
+            // Core EditorConfig Options
+            editorconfig.AppendLine("###############################");
+            editorconfig.AppendLine("# Core EditorConfig Options   #");
+            editorconfig.AppendLine("###############################");
```

Comment:
would prefer to not have extraneous stuff like the #####################.   The title though seems good :)

Reply:
I agree. Let's keep the title but remove the # lines.